server/service/src/sync/translations/requisition.rs

```diff
@@ -160,6 +160,47 @@ pub struct LegacyRequisitionRow {
     #[serde(deserialize_with = "empty_str_as_option_string")]
     pub programID: Option<String>,
 }
+
+/// When mSupply central creates transfers it copies over all of the data
```

Comment:
there is similar method that existing in invoice already, copied but added partial row struct because now there are two field to manual deserialise (so thought to do it with a struct)